diagnostic.md

```diff
@@ -45,7 +58,19 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+.big {
+
+}
+#contentPane {
+
+}
+.important.red {
+
+}
 
+div p {
```

Comment:
```div p``` will select all ```<p>```s that have ```<div>``` as an ancestor regardless of depth.  ```div > p``` selects ```<p>```s that have a ```<div>``` as a parent.